### PR TITLE
Home page mobile friendly

### PR DESCRIPTION
**Pull request by Mk996**
* **What kind of change does this PR introduce?**
* It is a bugfix PR. It fixes all the issues related to the home page's mobile-friendliness.

* **What is the new behavior (if this is a feature change)?**
Untitled

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
* No

* **List of fixes**:
* Main logo and description alignment issues.
* Hamburger menu alignment fixed.
* Horizontal scrolling due to overflowing elements fixed.
* Install text and command showing properly.
* Tab sections got a new mobile-friendly layout.
* Footer properly aligned and readable.

### CODE REVIEW

File: docs/source/_static/vendor/gallery.css [hunk 1]
```diff
@@ -59,6 +59,15 @@
     }
 }
 
+@media only screen and (max-width: 600px) {
```
Comment (Mk996) [inline docs/source/_static/vendor/gallery.css:62]: Done for stopping the overflowed horizontal scroll

File: docs/source/_static/vendor/installtext.css [hunk 2]
```diff
@@ -26,6 +26,13 @@
       font-size: 27px;
     }
   }
+
```
Comment (Mk996) [inline docs/source/_static/vendor/installtext.css:29]: To fit the the code section in the page

File: docs/source/_static/vendor/styles.css [hunk 3]
```diff
@@ -166,11 +167,27 @@ p {
 
     .hero-title {
         font-size: 4em;
+        margin-bottom: 2vh;
```
Comment (Mk996) [inline docs/source/_static/vendor/styles.css:170]: center the button on the main description

File: docs/source/_static/vendor/styles.css [hunk 4]
```diff
@@ -166,11 +167,27 @@ p {
...
+        width: 100vw;
+        justify-content: center;
     }
 
     .hero-logo {
         max-height: 60px;
     }
+
```
Comment (Mk996) [inline docs/source/_static/vendor/styles.css:178]: To align the logo on the mobile page

File: docs/source/_static/vendor/styles.css [hunk 5]
```diff
@@ -271,19 +289,17 @@ blockquote p::after {
...
-    #footer-columns {
-        display: flex;
-        flex-direction: column;
-        align-items: center;
-        text-align: center;
-    }
 
-    .footer-actions {
-        max-width: 50vw;
+    .footer-column ul.link-list li.link-list {
```
Comment (Mk996) [inline docs/source/_static/vendor/styles.css:293]: To align the footer in place

File: docs/source/_static/vendor/tabs.css [hunk 6]
```diff
@@ -44,12 +44,52 @@
     color: rgb(108, 122, 137);
 }
 
+@media screen and (max-width: 600px) {
```
Comment (Mk996) [inline docs/source/_static/vendor/tabs.css:47]: To provide the new look to the tabs section

File: docs/source/_templates/home.html [hunk 7]
```diff
@@ -592,11 +592,11 @@ <h1 class="gallery-title">Features</h1>
                                     <div>
                                         <a href="fury-pybullet.html"><img src="https://furiousatoms.com/_static/images/fullerene_1.png" alt="fullrene Modellin" align="middle"></a>
                                     </div>
-                                    <div class="row p-t-5">
-                                        <div class="col-sm">
+                                    <div class="row">
```
Comment (Mk996) [inline docs/source/_templates/home.html:595]: Appropriate Spacing in the molecular tab

File: docs/source/_templates/home.html [hunk 8]
```diff
@@ -697,10 +697,18 @@ <h1 class="gallery-title">Features</h1>
                             <h5>About</h5>
                         </div>
                         <ul class="link-list">
-                            <li class="link-list"><a class="footer-link" href="community.html#contributors"><i class="fa fa-angle-double-right"></i>Developers</a></li>
-                            <li class="link-list"><a class="footer-link" href="https://discord.gg/6btFPPj"><i class="fa fa-angle-double-right"></i>Support</a></li>
-                            <li class="link-list"><a class="footer-link" href="installation.html"><i class="fa fa-angle-double-right"></i>Download</a></li>
-                            <li class="link-list"><a class="footer-link" href="auto_tutorials/index.html"><i class="fa fa-angle-double-right"></i>Tutorials</a></li>
+                            <li class="link-list"><a class="footer-link" href="community.html#contributors"><i class="fa fa-angle-double-right"></i>
```
Comment (Mk996) [inline docs/source/_templates/home.html:700]: Giving space between the double arrow and text in the footer makes the layout consistent all over

**Review by skoudoro — APPROVED** [review]
Looks good @Mk996, Thank you. merging

**Pull request merged**